tests/functional/i/invalid/invalid_field_call.py

```diff
@@ -0,0 +1,59 @@
+"""Tests for the invalid-field-call message."""
+
+# pylint: disable=unused-import, invalid-name, missing-module-docstring, missing-class-docstring, missing-function-docstring, too-few-public-methods, unsubscriptable-object, multiple-statements
```

Comment:
```suggestion
# pylint: disable=missing-class-docstring, missing-function-docstring, too-few-public-methods
```

I think it is better to disable all others in line so we see what is wrong there.

Reply:
Took your suggestion (but for invalid-name) in 
3f2a279.